src/nvim/normal.c

```diff
@@ -2116,12 +2109,24 @@ static void op_function(oparg_T *oap)
     // function.
     virtual_op = MAYBE;
 
-    (void)call_func_retnr(p_opfunc, 1, argv, false);
+    (void)call_func_retnr(func, 1, argv, false);
 
     virtual_op = save_virtual_op;
   }
 }
 
+// Handle the "g@" operator: call 'operatorfunc'.
+static void op_function(oparg_T *oap)
+{
+  call_function("E774: 'operatorfunc' is empty", p_opfunc, oap);
+}
+
+// Handle the "g?" operator: call 'encodefunc'.
+static void enc_function(oparg_T *oap)
+{
+  call_function("E774: 'encodefunc' is empty", p_encfunc, oap);
```

Comment:
Even if you are not translating right here you must mark message for translation with `N_`. This is needed to automatically create a list of messages to translate. I would not suggest to go with E774 and two error messages where you can go with E774 and one error message: it is not like option name needs to be translated so you can have `emsgf(_("E774: '%s' is empty"), option_name)` rather then `EMSG(_(msg))`. (Do *not* use `EMSG2()`, ever since `emsgf()` was introduced it has no purpose.)